src/common/utils.jsx

```diff
@@ -53,6 +54,14 @@ export const formatDate = value => {
   return value.toLocaleDateString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit' });
 };
 
+export const showToast = (toastType, toastTitle, toastMessage, toastPosition) => {
+  return CustomToast({
+    icon: toastType,
+    title: toastTitle,
+    message: toastMessage,
+    position: toastPosition,
+  });
+};
```

Comment:
I don't think we are using this function anywhere? Can we please remove this?

Reply:
Actually instead of removing, can we use this function wherever it's needed?

Reply:
For some reason if I export that function and use it somewhere else it doesn't work.

Reply:
Ah I see, I think it has something to do with not being able to use hooks in non-React components. Let's just remove this function from here then.